Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/CorelliPowderCalibrationCreateTest.py

```diff
@@ -5,70 +5,134 @@
+            CorelliPowderCalibrationCreate(
+                InputWorkspace='perturbed', OutputWorkspacesPrefix='cal_',
+                TofBinning=[300, 1.0, 16666.7], PeakPositions=self.spacings_reference, SourceToSampleDistance=10.0,
+                FixY=False, ComponentList='bank1', ComponentMaxTranslation=0.03, ComponentMaxRotation=6,
+                Minimizer='differential_evolution', MaxIterations=10)
+        ConvertUnits(InputWorkspace='perturbed', Target='dSpacing', EMode='Elastic', OutputWorkspace='perturbed_dS')
+        results = CompareWorkspaces(Workspace1='test_workspace_dSpacing', Workspace2='perturbed_dS', Tolerance=0.001,
+                                    CheckInstrument=False)
+        DeleteWorkspaces(['perturbed_dS'])
+        return results.Result
+
+    def test_exceptions(self):
         # Both FixSource=True, AdjustSource=True can't be True
         try:
             CorelliPowderCalibrationCreate(
-                InputWorkspace='test_workspace', OutputWorkspacesPrefix='cal_',
-                TofBinning=[300, 1.0, 16666.7], PeakPositions=spacings_reference, FixSource=True, AdjustSource=True,
-                ComponentList='bank1', ComponentMaxTranslation=0.2, ComponentMaxRotation=10)
+                InputWorkspace='test_workspace_TOF', OutputWorkspacesPrefix='cal_',
+                TofBinning=[300, 1.0, 16666.7], PeakPositions=self.spacings_reference, FixSource=True,
+                AdjustSource=True, FixY=False, ComponentList='bank1', ComponentMaxTranslation=0.2,
+                ComponentMaxRotation=10)
         except RuntimeError as error:
             assert 'Some invalid Properties found' in str(error)
 
         # Both FixSource=True, AdjustSource=True can't be False
         try:
             CorelliPowderCalibrationCreate(
-                InputWorkspace='test_workspace', OutputWorkspacesPrefix='cal_',
-                TofBinning=[300, 1.0, 16666.7], PeakPositions=spacings_reference, FixSource=False, AdjustSource=False,
-                ComponentList='bank1', ComponentMaxTranslation=0.2, ComponentMaxRotation=10)
+                InputWorkspace='test_workspace_TOF', OutputWorkspacesPrefix='cal_',
+                TofBinning=[300, 1.0, 16666.7], PeakPositions=self.spacings_reference, FixSource=False,
+                AdjustSource=False, FixY=False, ComponentList='bank1', ComponentMaxTranslation=0.2,
+                ComponentMaxRotation=10)
         except RuntimeError as error:
             assert 'Some invalid Properties found' in str(error)
 
-        # The calibration algorithm will attempt to correct the position and orientation of the bank so that peak
-        # centers for all detectors in the bank (not just detector-ID 155) approach our reference values. As
-        # a result, the final position and orientation is not exactly perpendicular to the X-axis and positioned
-        # five meters away from the sample.
+    @unittest.skip("causes surpassing the timeout in the Jenkins servers")
```

Comment:
Running the full unit test (comment out the `@unittest.skip`) takes about 27 sec on the analysis cluster, which is way below the timeout limit on build server (5 min).  Am I missing something here?

Reply:
I think the build servers are not as quick as the analysis cluster. The first time I sent this PR, the test timed out at 300 seconds. Go figure!